# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libxcb/xcb.pc.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./xcb.pc.in.orig	2008-02-28 16:32:58.000000000 +0000
+++ ./xcb.pc.in	2008-02-28 16:33:11.000000000 +0000
@@ -6,7 +6,7 @@
 Name: XCB
 Description: X-protocol C Binding
 Version: @PACKAGE_VERSION@
-Requires.private: @NEEDED@
+Requires: @NEEDED@
 Libs: -L${libdir} -lxcb
 Libs.private: @LIBS@
 Cflags: -I${includedir}
